### DISCUSSION TITLE
Conductor-ui behind krakend

### DISCUSSION THREAD

**Discussion by witchbutter** [Netflix/conductor#3355] [Q&A]
Is it possible to connect to conductor-ui behind an api gateway that handles authentication?  There seems to be no documentation pertaining to this.

**Comment by v1r3n** [comment]
Hi @witchbutter this should be possible by putting an api-gateway that validates each request for authenticated user (via presence of token or other mechanism) .

At orkes, we use similar approach where each request is required to have a valid authentication header identifying the user.
